src/proto/message_translation/tbots_protobuf_test.cpp

```diff
@@ -124,41 +124,3 @@ TEST(TbotsProtobufTest, vision_msg_test)
                           robot, robot_states_map[robot.id()]);
                   });
 }
-
-TEST(TbotsProtobufTest, robot_state_test)
```

Comment:
maybe I missed it, but I think it's missing a test for World

Reply:
nvm I see it in another file

Reply:
I ended up adding tests for the translation functions in the classes test file (eg. `world_test.cpp`). And in the test I convert: our representation -> proto representation -> our representation and check if the two objects with our representation have the same properties. Since both translation functions are used in this process, I decided not to add tests to `tbots_protobuf_test.cpp`.